sanskrit_parser/rest_api/api_v1.py

```diff
@@ -24,7 +21,20 @@
                          default_label=api_blueprint.name,
                          prefix=URL_PREFIX, doc='/docs')
 
-analyzer = SanskritMorphologicalAnalyzer()
+analyzer = LexicalSandhiAnalyzer()
+
+
+def jedge(pred, node, label):
```

Comment:
Shouldn't these be combined with the corresponding functions in `datastructures.py`?

Reply:
I'll look, but there was some reason I left them separate. Possibly just a minor difference in behavior that can be absorbed into the other version.

Reply:
The difference is that one outputs in SLP1 and the other in devanagari (for the Web UI). I'll open an issue on this. Needs a bit of thought.